api/store.proto

```diff
@@ -0,0 +1,152 @@
+syntax = "proto3";
```

Comment:
Not sure I follow. This isn't used yet - it's being added so that events can use it.